src/components/landing/ActiveCell.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import AppIcon from "@/icons/AppIcon";
+import EditIcon from "@/icons/EditIcon";
+import { Announcement } from "@/models/Announcement";
+import { dateInRange, formatDate } from "@/utils/utils";
+import { useEffect, useState } from "react";
+
+interface Props {
+  announcement: Announcement;
+}
+
+export default function ActiveCell({ announcement }: Props) {
+  const [currentDate, setCurrentDate] = useState(new Date());
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      setCurrentDate(new Date());
+    }, 500);
+
+    return () => clearInterval(intervalId);
+  }, []);
+
+  return (
+    <div className="flex flex-col p-6 items-start md:items-end md:flex-row justify-center gap-6 md:gap-8 self-stretch bg-neutral-white rounded-lg border border-other-stroke relative">
+      <img
+        src={announcement.imageUrl}
+        className="h-[265px] md:w-[108px] md:h-[108px] self-stretch rounded-lg bg-lightgray object-cover bg-center"
```

Comment:
Why is there a light gray background? If this background is needed, make sure to use a color that's in our design system instead of this.